src/a11y-dialog.ts

```diff
@@ -212,17 +212,82 @@ function moveFocusToDialog(node: HTMLElement) {
   focused.focus()
 }
 
+// Elements with these ARIA roles make their children
+// `presentational`, which nullifies their semantics.
+// @see: https://www.w3.org/TR/wai-aria/
```

Comment:
Is that relevant to finding focusable children though?

Reply:
IMO, yes. It’s a common enough pattern for anchors and buttons to have child elements, and 

1. we may as well bail out instead of walking into their subtrees, plus,
2. if the author has for some reason done something like `a > a`, the browser won’t count the descendant `a` as tabbable and nor should we